inc/customizer/controls/react/src/icon-picker/Control.js

```diff
@@ -0,0 +1,8 @@
+import IconField from './IconField';
```

Comment:
Same thing here, the icon-picker should either be part of the `common` directory in the eventuality that it's going to be used at a later date in another control, or you can simply add it in the repeater folder. 

It doesn't interact with the customizer API as well, so no need for this wrapper component.